resources/js/src/app/components/itemList/filter/FindologicFilterWrapper.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <client-only>
+    <div
+      class="findologic-filter-wrapper"
+      :class="{ 'mb-5': facets.some((e) => e.isMain === true) }"
+      v-show="facets && facets.length > 0"
+    >
+      <div class="ml-0 main-filters">
+        <div
+          class="container-max component-loading page-content"
+          :class="{ isLoading: isLoading }"
+        >
+          <div class="card-columns row">
+            <client-only>
+              <div class="w-100">
+                <!-- SSR:template(findologic-item-filter) -->
+                <findologic-item-filter
+                  template-override="#vue-findologic-item-filter"
+                  v-for="facet in facets"
+                  :facet="facet"
+                  :key="facet.id"
+                  :filtersPerRow="filtersPerRow"
+                  :fallbackImageColorFilter="fallbackImageColorFilter"
+                  :fallbackImageImageFilter="fallbackImageImageFilter"
+                  :current-category="currentCategory"
+                  :show-category-filter="showCategoryFilter"
+                />
+
+                <!-- /SSR -->
```

Comment:
I think the SSR:template comment is only required with separate .twig and .js files. We should be able to remove it.

```suggestion
                <findologic-item-filter
                  template-override="#vue-findologic-item-filter"
                  v-for="facet in facets"
                  :facet="facet"
                  :key="facet.id"
                  :filtersPerRow="filtersPerRow"
                  :fallbackImageColorFilter="fallbackImageColorFilter"
                  :fallbackImageImageFilter="fallbackImageImageFilter"
                  :current-category="currentCategory"
                  :show-category-filter="showCategoryFilter"
                />
```